install/scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Versisn 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO UNDER DEVELOPMENT
+# This script serves as the entrypoint for starting Forseti server in a Docker container.
+# Ref. https://docs.docker.com/engine/reference/builder/#entrypoint
+#
+# This script assumes it's running in a container on a GCE Container Optimized OS VM
+# that is already authorized with the forseti service account.
+# gsutil will automatically use the hosting VM's service account.
+# TODO Other scenarios.
+
+# TODO named arg and arg validation
+# TODO arg to control which services to start (default to server for now)
+BUCKET=$1
+
+# Download config files
```

Comment:
Probably need to include the creation of the buckets in the entrypoint, if we are really automating this. @blueandgold please advice

Reply:
upon further research, I realized that Forseti installation does a lot more than just GCS bucket. Service account, role assignment, firewall rules, etc, all needs to be done during the installation. We definitely want to automate all these steps in the end, and need more discussion regarding what is the best way to automate. For a first step, I suggest that we keep this entrypoint file as is, and advice on using the manual instructions on forseti website to create the resounces.

Reply:
[instruction for manual resource creation](https://forsetisecurity.org/docs/latest/develop/dev/setup.html)

Reply:
My 2c. Keep the entrypoint focused on just starting Forseti services.

Keep all deployment as part of the CI\CD process.

Forseti runtime deployment of code is not ideal as would prefer to scan images for vulnerabilities before deployment.